tests/python/pants_test/pants_run_integration_test.py

```diff
@@ -327,6 +328,9 @@ def run_pants_with_workdir_without_waiting(self, command, workdir, config=None,
         ini.write(fp)
       args.append('--pants-config-files=' + ini_file_name)
 
+    if not Path(get_buildroot()).samefile(Path(".")):
+      shutil.copy(str(Path(get_buildroot(), "pyproject.toml")), str(Path(".")))
```

Comment:
Hm this is surprising and concerning to me. Are you sure it’s necessary? We don’t have to do this for any other tooling config files, and the change you made means that the pants.pex will already have the config in its chrooted folder.

Reply:
I'm sure it's necessary. It tripped me up for a good part of the morning because I had the same expectation you had. That's why I ended up just touching the file, but I still think this latest version is more correct.

Reply:
What happens if you include it as a dependency on the Pants.pex binary target like you have now AND the IT target? Not sure it will make a difference, but possibly. 

I gotta run to class and don’t want to block on this because we can always address in a followup, but would love to make sure there’s no way to avoid this.

Reply:
OK. Found how it works for pants.pex etc. Leads me to a slightly more elegant solution. Will push new commit. (Still a hack, but the same as others)

Reply:
See my latest commit (hopefully explains what's going on)